packages/dashboard/src/components/widgets/tile/tile.tsx

```diff
@@ -113,56 +125,62 @@ const WidgetTile: React.FC<WidgetTileProps> = ({
         borderRadius: borderRadiusBadge,
         backgroundColor: colorBackgroundContainerContent,
       }}
+      onMouseEnter={() => setShowActionButtons(true)}
+      onMouseLeave={handleOnMouseLeave}
     >
+      {enableActionButtons && (
+        <div className='tile-button-container'>
+          {widget.type !== 'text' && iotSiteWiseClient && (
+            <CSVDownloadButton
+              variant='inline-icon'
+              fileName={`${widget.properties.title ?? widget.type}`}
+              client={iotSiteWiseClient}
+              widgetType={widget.type}
+              queryConfig={
+                widget.properties.queryConfig as StyledSiteWiseQueryConfig
+              }
+            />
+          )}
+          <DeletableTileAction
+            variant='inline-icon'
+            handleDelete={handleDelete}
+          />
+          <ConfirmDeleteModal
+            visible={visible}
+            headerTitle='Delete selected widget?'
+            cancelTitle='Cancel'
+            submitTitle='Delete'
+            description={
+              <Box>
+                <Box variant='p'>
+                  Are you sure you want to delete the selected widget? You'll
+                  lose all the progress you made to the widget
+                </Box>
+                <Box variant='p' padding={{ top: 'm' }}>
+                  You cannot undo this action.
+                </Box>
+              </Box>
+            }
+            handleDismiss={handleCloseModal}
+            handleCancel={handleCloseModal}
+            handleSubmit={handleSubmit}
+          />
+        </div>
+      )}
+
       {headerVisible && (
         <div
           style={{
             display: 'flex',
             justifyContent: 'space-between',
             padding: `${spaceScaledXs} ${spaceScaledXs} ${spaceScaledXxs} ${spaceScaledM}`,
-            borderBottom: `2px solid ${colorBorderDividerDefault}`,
```

Comment:
Better to add this borderBottom until we completely remove the header and show the title in respective charts.